EXCEPTION LIST
The path 'foo' was not found in the trash directory '/tmp/nix-shell.v7e11e/safe-rm/functional/l3/.trash' despite being listed in the trash index. This can be fixed by manually deleting the entry from the index or deleting everything (i.e. sr e).

LOGS LIST
[2020-05-31 12:00:00][functional.getIndex][Debug][src/SafeRm.hs:194:4] Index path: <dir>/l3/.trash/.index.csv
[2020-05-31 12:00:00][functional.getIndex.readIndex][Debug][src/SafeRm/Data/Index.hs:103:4] Index path: <dir>/l3/.trash/.index.csv
[2020-05-31 12:00:00][functional.getIndex.readIndex.readIndexWithFold][Debug][src/SafeRm/Data/Index.hs:108:8] Found: MkPathData {pathType = PathTypeFile, fileName = MkPathI {unPathI = "foo"}, originalPath = MkPathI {unPathI = "<dir>/l3/.trash/foo"}, created = MkTimestamp {unTimestamp = 2022-09-28 02:58:33}}
[2020-05-31 12:00:00][functional][Error][src/SafeRm/Runner.hs:126:8] MkExceptionI (Proxy ExceptionIndex 'TrashPathNotFound) (MkPathI {unPathI = "<dir>/l3/.trash"},MkPathI {unPathI = "foo"})


